tasks/regular/modular_arithmetic.py

```diff
@@ -154,33 +154,29 @@ def __init__(self,
     self._operators = (OP_BY_CHARACTER[op] for op in operators)
 
   @functools.partial(jax.jit, static_argnums=(0, 2, 3))
-  def sample_batch(
-      self,
-      rng: jnp.ndarray,
-      batch_size: int,
-      sequence_length: int,
-  ) -> Mapping[str, jnp.ndarray]:
+  def sample_batch(self, rng: jnp.ndarray, batch_size: int,
```

Comment:
Please format the code properly here, with the line breaks and the comma at the end of arguments.

Reply:
I was following the formatting in the other files in `regular`. Just to confirm, should I still reformat the code?

Reply:
Yes, please!

Reply:
Done!